app/models/cda/court_application.rb

```diff
@@ -29,6 +31,22 @@ def defendant
       end
     end
 
+    def appeal?
+      application_category == "appeal"
+    end
+
+    def breach?
+      application_category == "breach"
+    end
+
+    def maat_reference
+      linked_maat_id unless linked_maat_id&.starts_with?(DUMMY_MAAT_PREFIX)
```

Comment:
That's totally fine,
If we want to make it more junior-friendly we could add a guard clause with if and explicit return:
```
  return nil if linked_maat_id&.starts_with?(DUMMY_MAAT_PREFIX)
  linked_maat_id
```
Still, this makes our life hard: we don't know if `linked_maat_id` is nil or if it is a dummy. In the past, cases like this were not easy to debug. (but I guess there is not much we can do for now)